### DISCUSSION TITLE
Port forwarding PART-DB

### DISCUSSION THREAD

**Discussion by dominykasLi** [Part-DB/Part-DB-server#797] [General]
Hello,

how safe would it be for me just to port-forward PART-DB on my home network? I am running on my old laptop Debian 17. Of course packages are updated.

The mission is to be able to reach the program through anywhere, but I do not want to pay third-party companies to host it.

**Comment by jbtronics** [comment]
Part-DB can be exposed to the internet. You should revoke all permissions for the anonymous user and use strong passwords, so that only authenticated users can actually do stuff.

You should set up HTTPS so that traffic is encrypted and you should keep everything up to date.
